### ISSUE TITLE
nn.Bottle support

### ISSUE THREAD

**Issue by brollb** [dfst/deepforge#845]

**Comment by brollb** [comment]
Updating the `nn` package exposed a minor bug in the parser; it was failing to resolve class name aliases such as:
```
local BN, parent = torch.class('nn.BatchNormalization', 'nn.Module')
```
The parser was expecting `BN` to be `BatchNormalization`; not a shortened alias
